libglvnd fails to build with xorgproto

##### Description:

`libglvnd` fails to build with `xorgproto 2018.1-1` (replaced `glproto-1.4.17-1` )
https://www.archlinux.org/packages/extra/any/xorgproto/

```bash
libglx.c:282:29: error: use of undeclared identifier 'X_GLXCreateContextAtrribsARB'; did you mean 'glXCreateContextAttribsARB'?
                            X_GLXCreateContextAtrribsARB, True);
                            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
                            glXCreateContextAttribsARB
libglx.c:265:19: note: 'glXCreateContextAttribsARB' declared here
static GLXContext glXCreateContextAttribsARB(Display *dpy, GLXFBConfig config,
                  ^
libglx.c:291:54: error: use of undeclared identifier 'X_GLXCreateContextAtrribsARB'; did you mean 'glXCreateContextAttribsARB'?
        vendor = CommonDispatchFBConfig(dpy, config, X_GLXCreateContextAtrribsARB);
                                                     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
                                                     glXCreateContextAttribsARB
libglx.c:265:19: note: 'glXCreateContextAttribsARB' declared here
static GLXContext glXCreateContextAttribsARB(Display *dpy, GLXFBConfig config,
                  ^
2 errors generated.
```
##### Additional info:

distro: Arch Linux
libglvnd: `r565.212861e`
side note: 
build finished successfully `libglvnd-git-r565.212861e + glproto-1.4.17-1`

--- a/src/GLX/libglx.c
+++ b/src/GLX/libglx.c
@@ -53,6 +53,16 @@
 #define GLX_MINOR_VERSION 4
 #define GLX_VERSION_STRING "1.4"
 
+/*
+ * Older versions of glxproto.h contained a typo where "Attribs" was misspelled.
+ * The typo was fixed in the xorgproto version of glxproto.h, breaking the API.
+ * Work around that here.
+ */
+#if !defined(X_GLXCreateContextAttribsARB) && \
+    defined(X_GLXCreateContextAtrribsARB)
+#define X_GLXCreateContextAttribsARB X_GLXCreateContextAtrribsARB
+#endif
+
 static glvnd_mutex_t clientStringLock = GLVND_MUTEX_INITIALIZER;
 
 /**
@@ -279,7 +289,7 @@ static GLXContext glXCreateContextAttribsARB(Display *dpy, GLXFBConfig config,
                 vendor = __glXGetDynDispatch(dpy, screen);
                 if (vendor == NULL) {
                     __glXSendError(dpy, BadValue, 0,
-                            X_GLXCreateContextAtrribsARB, True);
+                            X_GLXCreateContextAttribsARB, True);
                     return None;
                 }
             }
@@ -288,7 +298,7 @@ static GLXContext glXCreateContextAttribsARB(Display *dpy, GLXFBConfig config,
 
     if (vendor == NULL) {
         // We didn't get a GLX_SCREEN attribute, so look at the config instead.
-        vendor = CommonDispatchFBConfig(dpy, config, X_GLXCreateContextAtrribsARB);
+        vendor = CommonDispatchFBConfig(dpy, config, X_GLXCreateContextAttribsARB);
     }
 
     if (vendor != NULL && vendor->staticDispatch.createContextAttribsARB != NULL) {

